tests/test_models_variational.py

```diff
@@ -101,12 +101,10 @@ def myloss(parameters, circuit, target):
 @pytest.mark.parametrize(test_names, test_values)
 def test_vqe(backend, method, options, compile, filename):
     """Performs a VQE circuit minimization test."""
-    if backend.name == "pytorch":
-        pytest.skip("Skipping VQE test for pytorch backend.")
-    if (method == "sgd" or compile) and backend.name != "tensorflow":
+    if (method == "sgd" or compile) and (backend.name not in ("tensorflow", "pytorch")):
         pytest.skip("Skipping SGD test for unsupported backend.")
-    if method != "sgd" and backend.name == "tensorflow":
-        pytest.skip("Skipping scipy optimizers for tensorflow.")
+    if method != "sgd" and backend.name in ("tensorflow", "pytorch"):
+        pytest.skip("Skipping scipy optimizers for tensorflow and pytorch.")
```

Comment:
In this test as well, since we only test torch and tf, maybe rather than using the backend fixture do
``` python
@pytest.mark.parametrize(engine, [PyTorchBackend(), TensorflowBackend()])
def test_vqe(engine, ...):
```

Reply:
here we are testing tensorflow and pytorch for the sgd method but then we are also testing scipy optimizers for the other backends